client/app/bundles/course/assessment/submission/actions/history.js

```diff
@@ -0,0 +1,12 @@
+import actionTypes from '../constants';
+
+export function selectPastAnswers(questionId, answerIds) {
+  const sortedAnswerIds = answerIds.sort((a, b) => b - a);
```

Comment:
Just to be clear, here you are trying to sort selected past answers in reverse order of `created_at` timing, for display of answers? Using `answerIds` work for now but it's an implicit assumption, not sure if there are other ways of solving this.

Reply:
Yeah, the problem comes from material-ui's `SelectField` which doesn't maintain the order of the answerIds. Will change it up a bit to sort by the date.